.github/PULL-REQUEST.md

```diff
@@ -0,0 +1,31 @@
+### Description
+<!-- Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required 
+for this change. -->
+Fixes #(issue)
+
+### Type of Change
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+### How Has This Been Tested?
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your 
+test configuration. -->
+- [ ] Test A
+- [ ] Test B
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
```

Comment:
As I can understand about this section, It is based on testing in the webapp. Currently, We have not implemented any testing, Adding this will confuse the contributor.